routes/quote_routes.js

```diff
@@ -0,0 +1,124 @@
+var bodyparser = require('body-parser');
+var Quote = require('../models/Quote');
+var fs = require('fs');
+
+module.exports = function(router) {
```

Comment:
Tabbing is a little off for this assignment, no need to change but change it for future assignments.